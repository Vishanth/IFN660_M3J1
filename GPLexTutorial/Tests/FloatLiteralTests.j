1.1
.1
.1f
.1F
.1d
.1D
1.e-1
1.e+1
1.1E-1
1.1E+1
1e-1
1e+1
11E-1
11E+1