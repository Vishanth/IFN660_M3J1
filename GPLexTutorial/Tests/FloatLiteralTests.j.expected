FLOATLITERAL (1.1)
FLOATLITERAL (0.1)
FLOATLITERAL (0.1)
FLOATLITERAL (0.1)
FLOATLITERAL (0.1)
FLOATLITERAL (0.1)
FLOATLITERAL (0.1)
FLOATLITERAL (10)
FLOATLITERAL (0.11)
FLOATLITERAL (11)
FLOATLITERAL (0.1)
FLOATLITERAL (10)
FLOATLITERAL (1.1)
FLOATLITERAL (110)
EOF